flagship/src/visitor/DefaultStrategy.ts

```diff
@@ -245,11 +245,9 @@ export class DefaultStrategy extends VisitorStrategyAbstract {
   }
 
   protected fetchVisitorCampaigns (visitor: VisitorDelegate) :CampaignDTO[] {
-    if (!visitor.visitorCache || !visitor.visitorCache.data ||
-      !visitor.visitorCache.data.campaigns) {
+    if (!visitor?.visitorCache?.data?.campaigns || !Array.isArray(visitor.visitorCache.data.campaigns)) {
```

Comment:
I think the !isArray is enough, no ? If the first condition is falsy, the isArray will be false too

Reply:
yes, indeed. I think TS don't like it. I'll fix it